mw-webapp/src/component/table/Table.tsx

```diff
@@ -29,7 +28,7 @@ export const Table = () => {
     columns,
     getCoreRowModel: getCoreRowModel(),
   });
-
+  let keyCounter = 0;
```

Comment:
Hahaha) comon) https://react.dev/reference/react/useId

Reply:
omg, this shouldn't have been here, fail)
![image](https://github.com/tritonJS826/masters-way/assets/95077794/e532818c-adab-4c35-a2f6-0df56f500982)